src/content/docs/tips/shadcn.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Setting up Shadcn UI
+---
+
+import { Tabs, TabItem, Aside, Steps,LinkCard } from "@astrojs/starlight/components";
+import ExampleLinks from '~/components/ExampleLinks.astro'
+
```

Comment:
I think it would be good to explain the problem we are solving in this tip, i.e what is layed out in the working with interactive components section 

a quick explainer about the islands being seperate and needing to create our own react component that wraps all the shadcn components parts